hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/HLESurveyTransformationPipelineBuilder.scala

```diff
@@ -6,19 +6,32 @@ import com.spotify.scio.values.SCollection
 import org.broadinstitute.monster.common.{PipelineBuilder, StorageIO}
 import org.broadinstitute.monster.common.msg._
 import org.broadinstitute.monster.dogaging.jadeschema.table.{HlesDog, HlesOwner}
+import org.slf4j.LoggerFactory
 import upack.Msg
 
 object HLESurveyTransformationPipelineBuilder extends PipelineBuilder[Args] {
 
   implicit val msgCoder: Coder[Msg] = Coder.beam(new UpackMsgCoder)
 
   case class RawRecord(id: Long, fields: Map[String, Array[String]]) {
+    val logger = LoggerFactory.getLogger(getClass)
 
     def getRequired(field: String): String = fields(field).head
 
     def getOptional(field: String): Option[String] = fields.get(field).flatMap(_.headOption)
 
-    def getArray(field: String): Array[String] = fields.get(field).getOrElse(Array.empty)
+    def getBooleanOption(field: String): Option[Boolean] = {
```

Comment:
Could this short-circuit and just return `Boolean`? Instead of `map` below, you'd use `contains`. I think that will cover the most common pattern we need in the data

Reply:
(The most common pattern being a check to see if a Y/N question is "Yes" before extracting a bunch of other fields)

Reply:
Yeah, that makes sense. I had made it return an option for fields like `od_hispanic` that we have in the schema as non-required booleans. For that field,  it feels a little weirder to default to false. What do you think?

Reply:
@danxmoran  I simplified things with your suggestions, but left a "getBooleanOption" method for now to be used for fields like `od_hispanic`, where we don't want to default to false if the field was not filled out. Let me know what you think. I am happy to change it if you disagree.

Reply:
I think returning the `Option` optimizes for storing the result as a column, and returning a raw `Boolean` optimizes for using the Y/N to drive other transformations. Here there's 1 of each case, but I think the Y/N driving will be much more common overall.

Reply:
Having both options works for me!